OMNA/bcast/bcast-client-manufacturer-code-registry.html

```diff
@@ -38,7 +38,7 @@
   <div class="fl size100of100  justify mbl">  
```

Comment:
This file doesn't display properly. It looks different from the other copied registries.